src/ServiceStack.EventStore/Main/EventStoreFeature.cs

```diff
@@ -49,12 +52,21 @@ public EventStoreFeature(SubscriptionSettings subscriptionSettings, params Assem
             log = LogManager.GetLogger(GetType());
         }
 
+        public EventStoreFeature(ClusterVNode clusterNode, params Assembly[] assembliesWithEvents)
+            : this(new SubscriptionSettings(), assembliesWithEvents)
+        {
+            this.clusterNode = clusterNode;
+        }
+
         public async void Register(IAppHost appHost)
         {
             appSettings = appHost.AppSettings ?? new AppSettings();
 
             var connectionSettings = GetConnectionSettings();
-            var connection = EventStoreConnection.Create(connectionSettings.GetConnectionString());
+
+            var connection = clusterNode != null 
+                                ? EventStoreConnection.Create(connectionSettings.GetConnectionString())
+                                : EmbeddedEventStoreConnection.Create(clusterNode);
```

Comment:
That's something that I was thinking about. To have the EventStoreFeature 'knowing' about the integration tests isn't ideal. However, the alternative - to create the EventStoreConnection (as an EmbeddedEventStoreConnection outside of the Feature and pass it in - seems even less satisfactory since the Feature to my mind should be encapsulating the creation of and access to the connection to the EventStore instance.